docker/docker-compose-centos-dev.yml

```diff
@@ -0,0 +1,82 @@
+# Copyright 2016, EMC, Inc.
```

Comment:
Hi, @zhangy42 ,  I just found it removes DHCP & rabbitmq.
Where do you document the usage & the rational /purpose ? (sorry if I missed anything.)
is it applied to all CentOS , or specific customer's CentOS ?
I can understand the isc-dhcp-server has different config folder on CentOS than Ubuntu. but just I'm worried about just removing it , should we  add a new dhcp image which works for CentOS?